extensions/HyperVExtension/src/HyperVExtension/Strings/en-US/Resources.resw

```diff
@@ -357,4 +357,8 @@
     <value>Unable to start the virtual machine management service. The Hyper-V feature may have been recently enabled but a reboot has not been initiated. Please check that the service exists in the Windows Services snap-in if a reboot has already been done.</value>
     <comment>{Locked="{Hyper-V}", "{Windows}"} Error text to show the user that virtual machine management service can't be started by the extension</comment>
   </data>
+  <data name="MicrosoftLogoAltText" xml:space="preserve">
+    <value>Microsoft Logo</value>

```

Comment:
Oh so, these images actually change based on the image you select. E.g if you select ubuntu, the ubuntu image will show up. My suggestion would be, to use the Image name as the first word then logo. So something like:

`{0} Logo`

Then in VMGalleryCreationAdaptiveCardSession.cs you could do 

```C#
{ "DiskImageAltText", _stringResource.GetLocalized("ImageLogoAltText", galleryImage.Name);},
```

Reply:
Nice catch.  Changed.  Check the new video in the description.